src/utils/series.ts

```diff
@@ -1,11 +1,11 @@
 import type { Playlist, PlaylistItem } from '#types/playlist';
-import type { EpisodeWithSeason, Season, Series } from '#types/series';
+import type { Season, Series } from '#types/series';
 
 export const getFiltersFromSeries = (playlist: Playlist): string[] =>
-  playlist.playlist.reduce(
-    (filters: string[], item) => (item.seasonNumber && filters.includes(item.seasonNumber) ? filters : filters.concat(item.seasonNumber || '')),
-    [],
-  );
+  playlist.playlist
```

Comment:
Can you add a comment here? This code is hard to read and probably should have been commented already.